File: Task-Supervision-App/SB-Backend/Task-Supervision-App/src/main/java/com/app/model/Task.java
package com.app.model;

import java.time.LocalDateTime;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;

@Entity
@Data
public class Task {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	@Column(unique = true)
	private String taskTitle;
	private String taskDesc;
	private String status;
	private LocalDateTime createdAt=LocalDateTime.now();
	
	
	
}

File: Task-Supervision-App/Ang-Fronted/Task-Supervision-App/src/app/task.ts

export interface Task {
    id: number;
    taskTitle: string;
    taskDesc: string;
    status: string;
    createdAt:string
    // Add more properties as needed
  }
  
  export interface ReqTask {
    taskTitle: string;
    taskDesc: string;
    status: string;
    // Add more properties as needed
  }